CreeDictionary/API/models.py

```diff
@@ -354,7 +374,7 @@ def search(user_query: str) -> Tuple[str, List[SearchResult]]:
         matched_wordforms: Set[EntryKey] = set()
 
         # First, let's add all matched analyses:
-        for analysis in cree_results.keys():
+        for analysis in [result[0] for result in cree_results]:
```

Comment:
I'm not sure what `result[0]` is. I think this is the `Tuple[Analysis, NormatizedCree, Lemma]`. If that's the case, could you make the latter a named tuple with properties `analysis: Analysis; cree: NormatizedCree; lemma: Lemma`?

Reply:
Sure I'll name it CreeMatch what do you think

Reply:
or CreeResult, since the variable name is "cree_results"

Reply:
`CreeResult` 👍